renovate.json

```diff
@@ -14,5 +14,27 @@
       "groupName": "docker images",
       "matchFileNames": [".docker/**"]
     }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["^\\.deps/.*\\.yaml$"],
+      "matchStrings": [
+        "# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\\nversion: (?<currentValue>.*)"
+      ],
+      "datasourceTemplate": "{{{ datasource }}}",
+      "depNameTemplate": "{{{ depName }}}",
+      "versioningTemplate": "semver"
+    },
+    {
+      "fileMatch": ["^Makefile$"],
+      "matchStrings": [
+        "# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\\nCONTROLLER_TOOLS_VERSION \\?= (?<currentValue>.*)",
```

Comment:
This could be make a bit simpler, and more generic using advanced capture like here https://docs.renovatebot.com/modules/manager/regex/#advanced-capture :)